source/tutorials/batch-inference/index.rst

```diff
@@ -0,0 +1,161 @@
+.. meta::
+    :description: How to do Batch Inference
+
+Batch Inference Tutorial
+========================
+
+In this tutorial you will learn how to create and run a Batch Inference execution in Valohai. This execution will use TensorFlow 2.4.1 to run inference on ten unique images using a pre-trained MNIST model.
+
+.. admonition:: Prerequirements
+    :class: attention
+
+    For this tutorial you will need:
+
+    * Python 3
+
+    We're also going to need two files:
+
+    * a trained MNIST model
+    * some images to classify, packaged in a single zip file
+
+    To make it easy for you they are available here, no need to download them:
+
+    * Model: `<s3://valohai-public-files/tensorflow-batch-inference-tutorial/model.h5>`_
+    * Images: `<s3://valohai-public-files/tensorflow-batch-inference-tutorial/images.zip>`_
+
+    If you want to, you can train the required model by following the tutorial `here <https://www.tensorflow.org/tutorials/quickstart/beginner>`_.
+
+Running on Valohai
+------------------
+
+To easily run our batch inference on Valohai, we will use it to run our code from the very beginning.
+
+If you don't already have a Valohai account, go to `<https://app.valohai.com/>`_ to create one for yourself.
+
+Similarly if you don't already have the Valohai command-line client, install it:
+
+.. code-block:: bash
+
+  pip install valohai-cli
+
```

Comment:
I'd remove this and just include it in the requirements, like in some of the other tutorials

```suggestion

```